source/developer/interactive-dialogs.rst

```diff
@@ -66,6 +66,8 @@ Each dialog supports elements for users to enter information.
 - **text**: Single-line plain text field. Use this for inputs such as names, email addresses or phone numbers.
 - **textarea**: Multi-line plain text field. Use this field when the answer is expected to be longer than 150 characters. 
 - **select**: Message menu. Use this for pre-selected choices. Can either be static menus or dynamic menus generated from users and public channels of the system. For more information on message menus, see :doc:`the documentation <interactive-messages>`.
+- **bool**: Checkbox option. Use this for binary selection.
```

Comment:
Are you also interested in documenting this type?

Reply:
No, documentation about `bool` type had already been written.
https://docs.mattermost.com/developer/interactive-dialogs.html#checkbox-element

When writing about `radio type`, I noticed that `bool` is not here, so I just added it.

Reply:
Missed that, thanks for the explanation :+1: